### PR TITLE
Add minutes

### PR DESCRIPTION
**Pull request by teresanunez**

### CODE REVIEW

File: 20210120/MINUTES.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        - [ ] Users can break Sardana by messing with configuration in Tango DB, diagnostic script should be created to check that.
+	   No updates.
+        - [ ] Configure CI on buster (problems with test suite on Python 3.6): status update.
+	   Zibi will dedicate time to this. It could be an issue with the packages
+	   coming from the distribution. The problems with the test suite are solved updating
+	   tango. The CI part is related to github vs gitlab.
+
+5. Overview of current developments / PR distribution
+    - Problems for axis attributes with same name but different properties [#1436](https://github.com/sardana-org/sardana/issues/1436)
+      Not updates. Henry will do the programming.
```
Comment (reszelaz) [inline 20210120/MINUTES.md]:
```suggestion
      Not updates (due to the holidays period) Henrik and Zibi will do the pair-programming.
```

File: 20210120/MINUTES.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+     Zibi will check it.
+   - Preselect milestone scope
+
+   Solaris uses releases, the MBI will use them, so we decide that a release will be done.
+   The milestones should be solved.
+   https://github.com/sardana-org/sardana/milestone/8
+   New issues could be added.
+
+   The manual tests will be done in:
+   debian9, debian10, centos, windows, singularity (conda images - debian)
```
Comment (reszelaz) [inline 20210120/MINUTES.md]:
```suggestion
   debian9, debian10, centos, windows, singularity (wrappling a docker image build on top of the conda base image)
```

File: 20210120/MINUTES.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+      It does not work if sardana is reinitialized. It sounds like a bug.
+      Zibi will look into that.
+    
+6. Migration to gitlab
+    Travis will not support the public OS for free, we are alredy using the budget
+    of minutes. Tango will move to gitlab.
+    The migration to gitlab will also imply a migration of the documentation, gitlab pages could host it.
+    We could wait and see the experience of other projects.
+    Solaris and MaxIV have their own repos in gitlab, but they agree on waiting.
+    We agree on keeping github.
```
Comment (reszelaz) [inline 20210120/MINUTES.md]:
```suggestion
    We agree on keeping github for the moment at the same time not starting to use more github features (in order to not make the eventual future migration harder).
```

File: 20210120/MINUTES.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    - Step scans burst/points_per_step parameter [#1450](https://github.com/sardana-org/sardana/issues/1450)
+      Related to the point "How to make scans depending on the status of the beam" (previous discussion)
+    - pre- and post-move hooks not available in mv macro [#1471](https://github.com/sardana-org/sardana/issues/1471)
+      A propostion: add pre-mv and post-mv to mv, have sardana custom setting to call them in a mv or not.
+      Meanwhile work on general hooks with more granularity, add hooks to macros or groups of macros.
+    - Access motor name of a mv or scans.
+      It is proposed to add a motor attribute to all macros with pre/post-moves
+    - PseudoCounters accessing pseudocounters.
+      It does not work if sardana is reinitialized. It sounds like a bug.
+      Zibi will look into that.
```
Comment (reszelaz) [inline 20210120/MINUTES.md]:
```suggestion
      @dschick will try to reproduce the problem and report it as an issue. 
```

File: 20210120/MINUTES.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+	      but if there are multiple motors the backlash is done and the people think that ctrl-C does not work.
+	      They have investigated this and figured out that with one motor the flag for stopping is applied but in a group the flag is
+	      applied to the group and not to all the elements. A PR is done where the stop flag is passed to each element of
+	      the motor group. One has to decide the correct behaviour: Solaris suggest that a first ctrl-c would do the backlash
+	      but if a second one is executed the backlash would not be done (stop/abort difference).
+	      Daniel agrees with that and thinks it is the way the hardware does.
+	      Zibi suggests to accept the current PR but create an new issue for distinguishing between stop and abort
+	      (one/two ctrl-C)
+	      A discussion about the motor group devices arised, the possibility of avoiding the creation will be analysed by
+	      Zibi, it could be that they are not actually needed.
```
Comment (reszelaz) [inline 20210120/MINUTES.md]:
```suggestion
	      A discussion about the motor group devices arised, the possibility of avoiding the creation of the Tango devices was proposed by @tiagocoutinho (in the past it was already mentione in [sardana-org/sardana#1338](https://github.com/sardana-org/sardana/issues/1338#issuecomment-637646445).
```

**Review by reszelaz — APPROVED** [review]
Thanks Teresa for the minutes! Very complete as always! I have only some minor suggestions.

**Review by reszelaz — APPROVED** [review]
Also, if you could add the pending points from the [2020-12-16 meeting](https://github.com/sardana-org/sardana-followup/blob/master/20201216/MINUTES.md) that we have not discussed today to the point 4. This would be:

- [ ] Teresa will investigate how do they store the ROI configuration (if store at all)
- [ ] macro plots in spock do not appear until macro finishes #166 - try with QtSpock.
- [ ] Verify if Sardana axis/controller attribute are memorized by default.
- [ ] Hang scan due to the counter/timer controller timeout in StateOne() - How to deal with exceptions in the controllers: Review the docs?
- [ ] Decouple attributes default values from memorized values #1458
DESY, MAXIV agree on the changes; SOLARIS will answer later.
- [ ] Recent comment from the DESY users: using hooks may get complicated, even if they are nice feature - Clarify exactly what it means

**Review by cpt-majkel — APPROVED** [review]

**Comment by reszelaz** [comment]
@13bscsaamjad, could you please review the minutes?
Also the next meeting will be chaired by MaxIV. We agreed on 2021-02-18 at 14:00.
Cheers!

**Comment by 13bscsaamjad** [comment]
Looks good. Thanks for the heads up!
Cheers!

**Comment by reszelaz** [comment]
Many thanks to all. Merging.

**Pull request merged**